frontend/src/views/camp/navigation/desktop/NavTopbar.vue

```diff
@@ -6,8 +6,7 @@
       <v-btn :to="campRoute(camp)" text>
         <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-tent</v-icon>
         <span class="sr-only-sm-and-down">{{
-          camp.title
-            | loading($tc('views.camp.navigation.desktop.navTopbar.campIsLoading'))
+          $vuetify.breakpoint.lgAndUp ? camp.title : camp.shortname
```

Comment:
In the issue we said that the shortname should always be displayed. If we have place, we also show the title